src/config/dynamic-theme-fixes.config

```diff
@@ -4000,6 +4000,23 @@ body {
 
 ================================
 
+miktex.org
+
+INVERT
+.icon-bar
+
+CSS
+body,
+#site-preheader,
+#site-header,
+#site-postheader,
+#site-content,
+.site-footer {
+    background: none !important;
```

Comment:
```CSSS
background-image: none !important;
```

Let's not override the original colors as well and just remove the background-image.